app/util/mapIconUtils.js

```diff
@@ -643,7 +639,7 @@ export function drawCitybikeIcon(
   }
   let color = 'green';
   if (showAvailability) {
-    if (!bikesAvailable) {
+    if (bikesAvailable < 1) {
       color = 'red';
```

Comment:
bikesAvailable might be null, so don't remove the check for that

Reply:
The value usually is `-1`. But I'm adding that condition too.